wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WMultiFileWidget.java

```diff
@@ -208,16 +229,27 @@ public void setFileTypes(final String[] types) {
 
 	/**
 	 * Determines the file types accepted by this widget. Note that duplicates are not allowed and these are not case
-	 * sensitive. A file type may be one of the following:
+	 * sensitive.<br>
+	 * The file type(s) can be either:
 	 * <ul>
-	 * <li>The string audio/* (Indicates that sound files are accepted.)</li>
-	 * <li>The string video/* (Indicates that video files are accepted.)</li>
-	 * <li>The string image/* (Indicates that image files are accepted.)</li>
-	 * <li>A valid MIME type with no parameters (Indicates that files of the specified type are accepted.)</li>
-	 * <li>A string whose first character is a "." (U+002E) character (Indicates that files with the specified file
-	 * extension are accepted).</li>
+	 * <li>MIME type </li>
+	 * <li>Extension</li>
 	 * </ul>
-	 *
+	 * <u>MIME type</u>: it is <i>type/subtype</i>, where <i>type</i> is <b>text</b>, <b>image</b>, <b>application</b> etc,
+	 * and <i>subtype</i> is <b>plain</b>, <b>jpeg</b>, <b>*</b> etc. Some example MIME types are:
+	 * <ul>
+	 * <li><i>text/*</i> - indicates that all text files MIME types are accepted, 'text/html', 'text/plain' etc.</li>
+	 * <li><i>image/jpeg</i> - indicates that only jpeg image files are accepted.</li>
+	 * </ul>
+	 * Setting mime type is more reliable, as the contents of the file is validated against allowed list.
+	 * <br>
+	 * <u>Extension</u>: is the suffix at the end of the file name, succeeding a <i>"."</i> character. It indicates that files with 
+	 * specified extension is allowed. Some example extensions are: 
+	 * <ul>
+	 * <li><i>.txt</i> - indicates any files with extension <b>txt</b> are accepted.</li>
+	 * <li><i>.jpg</i> - indicates any files with extension <b>jpg</b> are accepted.</li>
+	 * </ul>
```

Comment:
If keeping this extended elucidation then fix the markup as per the comment above.

Reply:
ok